ktor.tree

```diff
@@ -162,6 +162,8 @@
         <toc-element toc-title="Deployments"
```

Comment:
Is the plan to do a separate one for Maven? If not, then maybe file should be named something more generic. Also, I think the title should reflect what the intention is as opposed to the name of the plugin, i.e. "Distributing Applications using Gradle" 

And lastly, I think the top-level TOC we should probably also rename to Application Distribution

Reply:
> Is the plan to do a separate one for Maven?

Yes, in the separate article.